apps/src/sites/studio/pages/home/teacher_homepage.js

```diff
@@ -1,25 +1,42 @@
 import $ from 'jquery';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import GradientNavCard from '@cdo/apps/templates/teacherHomepage/GradientNavCard';
+// import GradientNavCard from '@cdo/apps/templates/teacherHomepage/GradientNavCard';
```

Comment:
I'm going to need it later, so I left it commented for now to appease the linter.